Sources/Rendering/OpenGL/Actor/index.js

```diff
@@ -93,11 +107,10 @@ function vtkOpenGLActor(publicAPI, model) {
     if (prepass) {
       model.context.depthMask(true);
       publicAPI.activateTextures();
-    } else {
-      // deactivate textures
-      model.activeTextures.forEach((child) => {
-        child.deactivate();
-      });
+    } else if (model.activeTextures) {
+      for (let index = 0; index < model.activeTextures.length; index++) {
+        model.activeTextures[index].deactivate();
+      }
```

Comment:
Did you noticed that the `forEach` were bad compare to a simple for loop?

Another approach for fast looping is that:

```
let count = model.activeTextures.length;
while(count--) {
  model.activeTextures[count].deactivate();
}
```